src/components/NetlifyForm/index.jsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * NetlifyForm component consists of server related functionalities that is used posting data and also avoiding captchas.
+ * The base code that is used in both netlifyForm and index are implemented from https://github.com/jon-fm/react-ssg-netlify-forms-demo
+ */
+import React, { useState, useEffect } from "react";
+
+const NetlifyForm = ({
+  formName,
+  beforeSubmit,
+  afterSubmit,
+  contactFormValues,
+  children,
+}) => {
+  // Keep things well-hidden during runtime (and from bots. They are exposed for netlify build bots.
+  const [inNetlifyBuild, setInNetlifyBuild] = useState(true);
+  useEffect(() => {
+    setInNetlifyBuild(false);
+  }, []);
+
+  // Honeypot Input state for "Honeypot" field, which used Honeypot form field.
+  const [honey, setHoney] = useState("");
+  // For application/x-www-form-urlencoded, spaces are to be replaced by +, encodeURIComponent() replacement with an additional replacement of %20 with +. */
```

Comment:
`encodeURIComponent() replacement with an additional replacement of %20 with +. */` I didn't understand this. Also there is */ at the end of it

Reply:
lines below made it clear but I should be able to understand it in the comments

Reply:
The sentence was not clean enough, so I changed it and also add documentation link.